packages/astro-uxds/_content/patterns/sign-in.md

```diff
@@ -1,6 +1,6 @@
 ---
-tags: components
-path: /components/sign-in
+tags: patterns
+path: /patterns/sign-in
 date: Last Modified
 layout: components.template.njk
```

Comment:
Here is were I'm thinking we drop the iframe stuff and change this layout to match the others that are under /patterns.